service/service.hpp

```diff
@@ -145,114 +279,72 @@ namespace raven
-                    {
-                        {
-                            "CONFIG_CREATE",       [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->create_config(json_data, sock);
-                        }},
-                        {
-                            "CONFIG_LOAD",         [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->load_config(json_data, sock);
-                        }},
-                        {
-                            "CONFIG_UNLOAD",       [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->unload_config(json_data, sock);
-                        }},
-                        {
-                            "CONFIG_INCLUDE",      [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->include_config(json_data, sock);
-                        }},
-                        {
-                            "SETTING_UPDATE",      [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->update_setting(json_data, sock);
-                        }},
-                        {
-                            "SETTING_REMOVE",      [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->remove_setting(json_data, sock);
-                        }},
-                        {
-                            "SETTING_GET",         [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->get_setting(json_data, sock);
-                        }},
-                        {
-                            "ALIAS_SET",           [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->set_alias(json_data, sock);
-                        },
-                        },
-                        {
-                            "ALIAS_UNSET",         [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->unset_alias(json_data, sock);
-                        },
-                        },
-                        {
-                            "SUBSCRIBE_SETTING",   [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->subscribe_setting(json_data, sock);
-                        },
-                        },
-                        {
-                            "UNSUBSCRIBE_SETTING", [this](json::json &json_data, uvw::PipeHandle &sock) {
-                            this->unsubscribe_setting(json_data, sock);
-                        },
-                        }
-                    };
+    std::shared_ptr<uvw::Loop> uv_loop_{uvw::Loop::getDefault()};
+    std::shared_ptr<uvw::PipeHandle> server_{uv_loop_->resource<uvw::PipeHandle>()};
+    std::filesystem::path socket_path_{(std::filesystem::temp_directory_path() / "raven-os_service_albinos.sock")};
+    bool is_error_occured{false};
```

Comment:
Name doesn't sound English. "error_occurred" would sound better.

Reply:
Yes, could you apply the change ?

Reply:
Done.